lib/index.js

```diff
@@ -91,9 +91,29 @@ module.exports = React.createClass({
 		});
 	},
 	render: function () {
+		var ignoredAttrs = [
+			"alwaysShowCalendars", "applyClass", "autoApply",
```

Comment:
It's better to get all this items from lib/get-options.js to avoid code duplication.